### PR TITLE
Fix/padding utility

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@ghislaineguerin Please review CS changes for https://trello.com/c/uBa595qw/1941-css-padding-for-existing-utility-classes-1
Thx!

**Comment by ghislaineguerin** [comment]
@RV-Adrian let's try to make all utility numbers increments of 8px

**Comment by RV-Adrian** [comment]
@ghislaineguerin Sounds good, please confirm the change above.

File: src/core/utilities.scss [hunk 1]
```diff
@@ -98,6 +98,61 @@ All utilities have a prefix u_ added to them
...
+
+.padding-sm-vertical {  
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+/* Margin Utilities
+ */
+
+.add-left { margin-left: 10px !important; }
```
Comment (ghislaineguerin) [inline src/core/utilities.scss]: @RV-Adrian 8px instead of 10?

File: src/core/utilities.scss [hunk 2]
```diff
@@ -98,6 +98,61 @@ All utilities have a prefix u_ added to them
...
+.add-right { margin-right: 10px !important; }
+
+.remove-bottom { margin-bottom: 0 !important; }
+.small-bottom { margin-bottom: 3px !important; }
```
Comment (ghislaineguerin) [inline src/core/utilities.scss]: 4px instead of 3px

File: src/core/utilities.scss [hunk 3]
```diff
@@ -98,6 +98,61 @@ All utilities have a prefix u_ added to them
...
+.half-bottom { margin-bottom: 10px !important; }
+.add-bottom { margin-bottom: 20px !important; }
```
Comment (ghislaineguerin) [inline src/core/utilities.scss]: 16px instead of 20

File: src/core/utilities.scss [hunk 4]
```diff
@@ -98,6 +98,61 @@ All utilities have a prefix u_ added to them
...
+.add-bottom-double { margin-bottom: 40px !important; }
```
Comment (ghislaineguerin) [inline src/core/utilities.scss]: 32 px instead of 40

File: src/core/utilities.scss (hunk 1 above)
Comment (RV-Adrian) [inline src/core/utilities.scss]: @ghislaineguerin Just tackling padding here. Margin in the next card (which we started to review, I think that caused some confusion)

File: src/core/utilities.scss (hunk 2 above)
Comment (RV-Adrian) [inline src/core/utilities.scss]: @ghislaineguerin Same as above

**Comment by ghislaineguerin** [comment]
👍

**Pull request merged**